apps/dashboard/app/controllers/files_controller.rb

```diff
@@ -132,9 +128,9 @@ def update
   # POST
   def upload
     upload_path = uppy_upload_path
-    @path = PosixFile.new(upload_path)
 
-    AllowlistPolicy.default.validate!(@path)
+    parse_path(upload_path)
+    validate_path!
```

Comment:
Since we put these into a `before_action`, can you add the `except :upload` to it?

Reply:
Added.